AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -1646,6 +1647,21 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		2AC85249A2221047409FABBB /* Embed Pods Frameworks */ = {
```

Comment:
Can you remove this change?  I think it may be added by a newer version of Cocoapods, but we can't move off of 0.37.2 yet due to compatibility issues.  Looking forward to those guys finally getting the 1.0 out.